frontend/ui/widget/touchmenu.lua

```diff
@@ -605,10 +605,11 @@ function TouchMenu:updateItems()
     self.page_info_left_chev:enableDisable(self.page > 1)
     self.page_info_right_chev:enableDisable(self.page < self.page_num)
 
-    local time_info_txt = os.date("%H:%M").." – "
-    local twelve_hr = G_reader_settings:isTrue("twelve_hour_clock")
-    if twelve_hr then
+    local time_info_txt
+    if G_reader_settings:isTrue("twelve_hour_clock") then
         time_info_txt = os.date("%I:%M %p").." – "
+    else
+        time_info_txt = os.date("%H:%M").." – "
     end
     time_info_txt = time_info_txt..Device:getPowerDevice():getCapacity().."%"
```

Comment:
Not as per my full suggestion :)
It jumps at me that both have `.." – "` , while having it on the next common line would make it clearer and less risky of desync.